components/widgets/resources/src/main/resources/resources/uicomponents/widgets/exportData.js

```diff
@@ -386,7 +386,10 @@ document.observe('xwiki:dom:loading', function() {
   }
 
   document.observe('xwiki:multisuggestpicker:selectionchanged', function() {
-    document.fire('xwiki:livetable:patients:filtersChanged');
+    var tableName = $$('.xwiki-livetable')[0] ? $$('.xwiki-livetable')[0].id.replace(/-.*/, '') : '';
```

Comment:
If you get the ID directly from the table, then there's no need for the `replace`.